lib/timeline/component/LineGraph.js

```diff
@@ -470,31 +485,38 @@ LineGraph.prototype._updateAllGroupData = function () {
       }
     }
 
-    //Update legendas, style and axis
-    for (var groupId in groupsContent) {
-      if (groupsContent.hasOwnProperty(groupId)) {
-        if (groupsContent[groupId].length == 0) {
-          if (this.groups.hasOwnProperty(groupId)) {
-            this._removeGroup(groupId);
-          }
-        } else {
-          var group = undefined;
-          if (this.groupsData != undefined) {
-            group = this.groupsData.get(groupId);
-          }
-          if (group == undefined) {
-            group = {id: groupId, content: this.options.defaultGroup + groupId};
-          }
-          this._updateGroup(group, groupId);
-          this.groups[groupId].setItems(groupsContent[groupId]);
+    const groupsData = this.groupsData._data || [];
+    for (let groupId in groupsData) {
+      if (groupsData.hasOwnProperty(groupId)) {
+        let group = undefined;
+        if (this.groupsData != undefined) {
+          group = this.groupsData.get(groupId);
+        }
+        if (group == undefined) {
+          group = {id: groupId, content: this.options.defaultGroup + groupId};
         }
+        this._updateGroup(group, groupId);
+        this.groups[groupId].setItems(groupsContent[groupId]);
       }
     }
+    this._calculateHeights();
     this.forceGraphUpdate = true;
     this.body.emitter.emit("_change",{queue: true});
   }
 };
 
+LineGraph.prototype._calculateHeights = function() {
+  let totalHeight = 0;
+
+  this.groupsData.forEach(d => {
+    totalHeight += d.rowHeightId[d.className];
```

Comment:
d.rowHeightId can be undefined.

![image](https://user-images.githubusercontent.com/15340567/92766914-050dd980-f36d-11ea-81ed-726786422881.png)